src/amo/components/Routes/index.js

```diff
@@ -46,6 +46,11 @@ const Routes = ({ _config = config }: Props = {}) => (
       path="/:lang/:application/addon/:addonSlug/reviews/"
       component={AddonReviewList}
     />
+    <Route
+      exact
+      path="/:lang/:application/review/:reviewId"
+      component={AddonReviewList}
```

Comment:
Did you consider making a new `FeaturedAddonReview` component? The new component could render `<AddonReviewList>` underneath the featured review. This separation of concerns might make the logic easier to reason about.

Reply:
Yes, I wasn't too happy about how much I had to change `AddonReviewList` to support this, so I'll give that a try.

Reply:
It got complicated trying to do that because the current list also contains a summary card. I did extract the Featured review into its own component and that seems to have simplified things a bit.